src/pages/Detail/Detail.js

```diff
@@ -1,7 +1,251 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import * as S from './StyleDetail';
+import Calendar from './component/DayPicker';
+import TimePicker from './component/TimePicker';
+import Map from './component/Map';
 
 const Detail = () => {
-  return <div>Detail</div>;
+  const [selectedDate, setSelectedDate] = useState(new Date());
+  const [isModalShow, setIsModalShow] = useState(false);
+  const [isCountModalShow, setIsCountModalShow] = useState(false);
+  const [detailData, setDetailData] = useState({});
+  const [time, setTime] = useState([]);
+  const [count, setCount] = useState(0);
+
+  useEffect(() => {
+    fetch('/data/detailData.json')
+      .then(res => res.json())
+      .then(detailData => setDetailData(detailData.description[0]));
+  }, []);
+
+  const handleModalBtn = () => {
+    setIsModalShow(!isModalShow);
+  };
+
+  const handleCountModal = () => {
+    setIsCountModalShow(!isCountModalShow);
+  };
+
+  const ShowTime = () => {
+    if (time.length === 0) {
+      return '시간선택';
+    } else if (time[1] === null) {
+      return `${time[0]}:00 ~ ${time[0] + 1}:00`;
+    } else {
+      return `${time[0]}:00 ~ ${time[1] + 1}:00`;
+    }
+  };
+
+  const PostTime = () => {
+    if (time[1] === null) {
+      console.log([time[0], time[0] + 1]);
+    } else {
+      console.log([time[0], time[1] + 1]);
+    }
+  };
+
+  const handleCountUp = () => {
+    setCount(count + 1);
+  };
+
+  const handleCountDown = () => {
+    setCount(count - 1);
+  };
```

Comment:
```suggestion
  const handleCount = (value) => {
      if (count + value === 0 && count + value === 10) return
      setCount(count + value)
  };
```
하나의 함수를 만들어 놓고 value 자리에 -1 혹은 1을 전달해서 숫자 증감을 할 수 있습니다.
그리고 count 가 1일때, count가 9일때 증감이 되지 않도록 하기 위해 위와 같은 조건을 줄 수 있겠습니다.